models/ges/ges/config.py

```diff
@@ -24,8 +28,16 @@ def config(
 
     # If the same variable is defined also as an environment variable, have that
     # override the value in the file.
-    for key in conf_dict.keys():
```

Comment:
I think we should keep this part, this is what allows overriding any of the values in `config.ini` with environment variables. The function app makes use of it, by setting things like `num_data_points` using envvars.

Reply:
Come to think of this now, would the following work: Make the variables set in `config.ini`/through envvars be the ones that `.secrets` uses, so `sql_host`, `sql_username`, `sql_pass, `sql_dbname`. So then they can be set in either `config.ini` or envvars and the user doesn't have to care about the fact that psycopg2 has some strange requirements. And then, at the end of this function, we would just do `conf_dict["user"] = conf_dict["sql_username"] @ conf_dict["sql_host"]` etc. (knowing that by the time we reach the end of this function, those values have to be in `conf_dict`, because they are in `config.ini`).

I'm thinking on the fly here, let me know if you think this isn't a good idea for some reason.